src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -157,7 +158,9 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
         #endregion
 
         #region IMemberContainer
-        public virtual IMember GetMember(string name) => GlobalScope.Variables[name]?.Value;
+        public virtual IMember GetMember(string name)
+            => GlobalScope.Variables[name]?.Value ?? this.GetSubmodule(name);
```

Comment:
Is it possible for `Value` to be null on purpose such that we'd be better off doing a try-get from `Variables`, then resorting to the module name?

Reply:
Unusual (normally it would be `Unknown`) but I guess it does not hurt to handle.

Reply:
Filtering changes since.